### PR TITLE
Advance with resource User

### PR DESCRIPTION
**Pull request by Linzeur**
Test with Get done

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :tweets
+  has_many :follows
```
Comment (deyviconde) [inline app/models/user.rb]: Follow accepts nil?

File: app/models/user.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class User < ApplicationRecord
+  has_many :tweets
```
Comment (deyviconde) [inline app/models/user.rb]: Tweet accepts nil?

**Comment by deyviconde** [comment]
cool

File: app/models/user.rb (hunk 1 above)
Comment (Linzeur) [reply app/models/user.rb]: Not, always it is necessary put values

**Pull request merged**